File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.java
import java.util.*;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        long N = sc.nextLong();
        long cnt = 0;
        long number = 665;
        String str;

        while(cnt < N){
            number++;
            str = Long.toString(number);
            if(str.contains("666")){
                cnt++;
            }
        }

        System.out.println(number);
    }
}

File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.kt
import java.io.*
import java.util.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))){

    val N = readLine().toInt()
    var cnt = 1
    var currentNum : Long = 666
    close()

    while(N > cnt) {

        currentNum++
        if(currentNum.toString().contains("666")) {
            cnt++
        }
    }

    val bw = BufferedWriter(OutputStreamWriter(System.out))
    bw.write(currentNum.toString())
    bw.flush()
    bw.close()
}